zapisy/apps/enrollment/courses/models/semester.py

```diff
@@ -25,6 +25,12 @@ class Semester(models.Model):
         blank=True,
         verbose_name='Czas otwarcia zapisów',
         help_text='Godzina powinna być ustawiona na 00:00:00, by studenci mieli otwarcie między 10:00 a 22:00.')
+    records_spacing = models.IntegerField(
+        null=True,
+        blank=True,
+        default=settings.ECTS_BONUS,
```

Comment:
```suggestion
        default=2,
```
To jest chyba _overkill_, i nie bardzo ma szansę działać zgodnie z intencją (_vide_ to, że w migracji znalazła się już podstawiona wartość). Trzymanie tej wartości w `settings.py` jest po to, żeby było łatwo nią sterować podczas uruchamiania aktualizacji czasów otwarcia, teraz będzie się to robić z poziomu ustawień semestru w panelu administracyjnym, i to wystarczy. W konsekwencji można będzie ją też wyrzucić z `settings.py`.

Reply:
Zastosowałem tę zmianę